draft-ietf-taps-interface.md

```diff
@@ -1681,28 +1703,33 @@ when it is temporarily not ready to receive messages.
 Connection -> Received<messageData, messageContext>
 ~~~
 
-A Received event indicates the delivery of a complete Message. It contains two objects,
-the received bytes as messageData, and the metadata and properties of the received
-Message as messageContext. See {{receive-context}} for details about the received context.
+A Received event indicates the delivery of a complete Message. It contains two
+objects, the received bytes as messageData, and the metadata and properties of
+the received Message as messageContext.
+
+The messageData object provides access to the bytes that were received for this
+Message, along with the length of the byte array.
 
-The messageData object provides access to the bytes that were received for this Message,
-along with the length of the byte array.
+The messageContext is provided to retrieve metadata about the message, see
+{{msg-ctx}}, and to refer to the message later on, e.g., to enable to map
```

Comment:
nit: I think this would read better as "to enable mapping" (to reduce the number of "to" occurrences in the sentence). But anyway, I'd rather not repeat the lower-level details here and instead say: "The messageContext is provided to retrieve metadata about the message, see {{msg-ctx}}, and to refer to the message later on, e.g., to map responses to prior requests."   That's just a suggestion, it's ok for me either way.